src/test/python/agents/atba/atba.py

```diff
@@ -52,7 +46,7 @@ def get_output_vector(self, game_tick_packet):
             color = self.renderer.create_color(121, 121, 0, 121)
             color2 = self.renderer.create_color(255, 0, 255, 255)
             self.renderer.draw_line_2d(100, 100, 1000, 500, self.renderer.black())
-            self.renderer.draw_rect_2d(0, 0, 1000, 1000, True, color)
+            self.renderer.draw_rect_2d(0, 0, 50, 50, True, color)
```

Comment:
Are we not getting rid of the default rendering in ATBA? It's a little obnoxious imo. We can disable it by default but leave the code in for reference

Reply:
It's a bit obnoxious but keep in mind that bot makers will never see this since it's in the test directory.

Reply:
Ok, that's fine. But the default config still has ATBA as a player for higher than P2. I understand flatBot replaced it, though. Just worried about first time users :^) Otherwise this is fine as a test.

Reply:
Maybe I'll make rendering and quick chatting part of atba's config and turn them both off by default.